data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+			`	After the audience is seated, an announcer once again comes on the loudspeaker. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting, and the next event will be starting shortly. As you sit waiting among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games.`
+			`	Half an hour passes before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. The members of each team are introduced to the crowd and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
+			`	The game starts. It's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience, but the nets are sturdy, and prevent any accidents.`
+			`	Eventually, a sphere finds its way into a hole, which lights up with the color of the team who scored: red. Much of the crowd jumps up and screams. You're not sure if everyone is a fan of the red team or if they're cheering simply because a point was scored. The sphere is brought back up automatically via a tube close to the hole and it's back in the game.`
+			`	The game continues, and each team accumulates a few points. The blue team, leading, scores a fifth point, and the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The Kimek with the translator explains what is happening.`
+			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, going from the team that scored the least all the way to the blue team at the end. Between the two events, Ulmodi is placed third. He and the rest of the top twenty contestants celebrate by running laps around the arena.`
+			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. The crowd begins to file out. You get back to where the train dropped you off and see that another one has arrived on the opposite side. A Heliarch agent explains that it is headed to the final arena where the cooperative game will take place.`
```

Comment:
After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena, at which point the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`